<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcessTool</class>
 <widget class="QWidget" name="ProcessTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#ProcessTool{
	background:#FFF
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QListWidget" name="list">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">#list{
	border:0;
}</string>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::InternalMove</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/add.png</normaloff>:/icons/add.png</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="toolTip">
    <string>Add a process</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/remove.png</normaloff>:/icons/remove.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete a process</string>
   </property>
  </action>
  <action name="actionModify">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/edit.png</normaloff>:/icons/edit.png</iconset>
   </property>
   <property name="text">
    <string>Modify</string>
   </property>
   <property name="toolTip">
    <string>Modify the process</string>
   </property>
  </action>
  <action name="actionEmpty">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/delete.png</normaloff>:/icons/delete.png</iconset>
   </property>
   <property name="text">
    <string>Empty</string>
   </property>
   <property name="toolTip">
    <string>Empty the process list</string>
   </property>
  </action>
  <action name="actionMark_as_unprocessed">
   <property name="text">
    <string>Un-mark</string>
   </property>
   <property name="toolTip">
    <string>Remove the 'processed' mark</string>
   </property>
  </action>
  <action name="actionSave_List">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save List</string>
   </property>
   <property name="toolTip">
    <string>Save this process list</string>
   </property>
  </action>
  <action name="actionOpen_List">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/folder.png</normaloff>:/icons/folder.png</iconset>
   </property>
   <property name="text">
    <string>Open List</string>
   </property>
   <property name="toolTip">
    <string>Open a process list</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
